js/tiles.js

```diff
@@ -273,12 +282,6 @@ const door = {
           this.wallRight = this.createImage("tiles/rooms/wall/rightChess.svg");
           this.ground = this.createImage("tiles/rooms/floor/banner-Chess.svg");
       },
-      visit: function() {
```

Comment:
You remove this alert box. Although I am for removing alerts, please motivate this, tell us why so the people who added this, understand that the code is better removed.